### DISCUSSION TITLE
Can I use bento docker image to run on standalone server without Yatai ?

### DISCUSSION THREAD

**Discussion by SpeedEX** [bentoml/BentoML#3227] [General] [answered]
I would like to use image from `bentoml containerize` command to run on another server without Yatai. I try to search but the document and tutorial seem to descript things around Yatai.

**Comment by aarnphm** [comment, marked as answer]
You can definitely run the container image after running `containerize`. Containerize is independent of Yatai. You can see here https://docs.bentoml.org/en/latest/concepts/deploy.html#containerize-bentos.

To run the container you can just use `docker run`.
